.github/workflows/fossa.yml

```diff
@@ -0,0 +1,35 @@
+name: FOSSA
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x]
+
+steps:
+  - uses: actions/checkout@v2
+  - uses: borales/actions-yarn@v2.0.0
+    with:
+      cmd: install
+  - uses: borales/actions-yarn@v2.0.0
+    with:
+      cmd: build
+  - uses: borales/actions-yarn@v2.0.0
+    with:
+      cmd: test
```

Comment:
install, build and test are run in a separate workflow, so let's not duplicate those here. Looking at the suggestion from @Fox32 it seems like they're not needed anyway